app/models/report.rb

```diff
@@ -15,4 +17,22 @@ class Report < ApplicationRecord
             length: {maximum: Settings.validates.report.max_length}
   validates :free_comment,
             length: {maximum: Settings.validates.report.max_length}
+
+  scope :recent_reports, ->{order created_at: :desc}
+  scope :active_reports, ->{where deleted: false}
+
+  class << self
+    def filter_report date, status
+      case
+        when date.present? && status.present?
+          where("date(created_at) = ? AND status = ?", date, status)
```

Comment:
nếu nhiều tham số thì e viết kiểu này cho dễ kiểm soát, ? sau nhiều quá thì ko ngồi đếm được 
```suggestion
          where("date(created_at) = :date AND status = :status", date: date, status: status)
```

Reply:
Vâng ạ

Reply:
Khi chuyển hàm này sang when case xong em chạy lại rubocop thì nó báo trường hợp này nên dùng if else nên em chuyển lại sang if else ạ.